db/change_cache.go

```diff
@@ -240,7 +223,7 @@ func (c *changeCache) EnableChannelIndexing(enable bool) {
 }
 
 // Inserts pending entries that have been waiting too long.
-func (c *changeCache) InsertPendingEntries(ctx context.Context) {
+func (c *changeCache) InsertPendingEntries(ctx context.Context) error {
```

Comment:
Add a note in the comment above to explain why error is included in the signature.